src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -462,18 +456,21 @@ void AddNativeLibrariesFromAssemblies (ZipArchive apk, string supportedAbis)
 						continue;
 					var data = ressozip.GetResourceData ();
 					using (var ms = new MemoryStream (data)) {
-						using (var zip = new ZipArchive (ms, ZipArchiveMode.Read)) {
-							foreach (var e in zip.Entries.Where (x => abis.Any (a => x.FullName.Contains (a)))) {
-								if (e.IsDirectory ())
+						Console.WriteLine ($"AddNativeLibrariesFromAssemblies {ressozip.Name}");
```

Comment:
Why are we `Console.WriteLine()`ing this? (Debug-spew?)

If we *do* want to write this, shouldn't this use some `Log` method, not `Console.WriteLine()`?